WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -16,12 +16,16 @@ open class QuickStartTourGuide: NSObject {
 
     private override init() {}
 
-    func setup(for blog: Blog) {
+    func setup(for blog: Blog, withCompletedSteps steps: [QuickStartTour] = []) {
         didShowUpgradeToV2Notice(for: blog)
 
 
         let createTour = QuickStartCreateTour()
         completed(tour: createTour, for: blog)
+
+        steps.forEach { (tour) in
+            completed(tour: tour, for: blog)
```

Comment:
I ended up using the basic idea, from what @Tug put together in https://github.com/wordpress-mobile/WordPress-iOS/pull/15406/files#r532550464 however I changed it up slightly to provide an option for the site creator to indicate what steps were completed.